tokens/src/core/global/typography.json

```diff
@@ -2,90 +2,284 @@
+        },
+        "h1": {
+          "base": {
+            "source": "$h1-font-size",
+            "$value": "2.5rem",
+            "$description": "Base font size of heading level 1."
+          },
+          "mobile": {
+            "source": "$h1-mobile-font-size",
+            "$value": "2.25rem",
+            "$description": "Mobile font size of heading level 1."
+          }
+        },
+        "h2": {
+          "base": {
+            "source": "$h2-font-size",
+            "$value": "2rem",
+            "$description": "Font size of heading level 2."
+          },
+          "mobile": {
+            "source": "$h2-mobile-font-size",
+            "$value": "{typography.font.size.h2.base}",
+            "$description": "Mobile font size of heading level 2."
+          }
+        },
+        "h3": {
+          "base": {
+            "source": "$h3-font-size",
+            "$value": "1.375rem",
+            "$description": "Font size of heading level 3."
+          },
+          "mobile": {
+            "source": "$h3-mobile-font-size",
+            "$value": "{typography.font.size.h3.base}",
+            "$description": "Mobile font size of heading level 3."
+          }
+        },
+        "h4": {
+          "base": {
+            "source": "$h4-font-size",
+            "$value": "1.125rem",
+            "$description": "Font size of heading level 4."
+          },
+          "mobile": {
+            "source": "$h4-mobile-font-size",
+            "$value": "{typography.font.size.h4.base}",
+            "$description": "Mobile font size of heading level 4."
+          }
+        },
+        "h5": {
+          "base": {
+            "source": "$h5-font-size",
+            "$value": ".875rem",
+            "$description": "Font size of heading level 5."
+          },
+          "mobile": {
+            "source": "$h5-mobile-font-size",
+            "$value": "{typography.font.size.h5.base}",
+            "$description": "Mobile font size of heading level 5."
+          }
+        },
+        "h6": {
+          "base": {
+            "source": "$h6-font-size",
+            "$value": ".75rem",
+            "$description": "Font size of heading level 6."
+          },
+          "mobile": {
+            "source": "$h6-mobile-font-size",
+            "$value": "{typography.font.size.h6.base}",
+            "$description": "Mobile font size of heading level 6."
+          }
+        },
+        "display": {
+          "1": {
+            "source": "$display1-size",
+            "$value": "3.75rem",
+            "$description": "Font size for heading of level 1."
+          },
+          "2": {
+            "source": "$display2-size",
+            "$value": "4.875rem",
+            "$description": "Font size for heading of level 2."
+          },
+          "3": {
+            "source": "$display3-size",
+            "$value": "5.625rem",
+            "$description": "Font size for heading of level 3."
+          },
+          "4": {
```

Comment:
On a local checkout of the docs site from this branch, it looks like the migrate of typography example tables to use `DataTable` is inheriting its `overflow: hidden` on table cells, cutting off the text of "Display 4" on smaller screen sizes. Probably not a big deal, but figured I'd flag it as it could be perceived as a style bug potentially.

<img width="1906" alt="image" src="https://github.com/user-attachments/assets/ea89a028-7914-42d5-b6e5-c46f1b5fbced">

<img width="196" alt="image" src="https://github.com/user-attachments/assets/48811055-b59e-416b-a73a-f445cdb1971f">

Reply:
[inform] Implemented in follow-up commit: [`6676868` (#3186)](https://github.com/openedx/paragon/pull/3186/commits/6676868e6c46d70d91f0bdc42b7c2848e3b0aa7c)